src/components/SearchInput/SearchInput.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { SearchInput } from './SearchInput'
+
+export default {
+  title: 'SearchInput',
+  parameters: {
+    info: `
+USWDS 2.0 SearchInput component
+
+Source: https://designsystem.digital.gov/components/header/
+`,
+  },
+}
+
+const mockSubmit = (): void => {
+  /* mock submit fn */
+}
+
+export const defaultSearchInput = (): React.ReactElement => (
+  <SearchInput onSubmit={mockSubmit}></SearchInput>
```

Comment:
a general formatting note: components that don't have `children` should use a self-closing tag (i.e., `<SearchInput />`)

you can also omit the `={true}` from props with a `true` value (i.e., `<SearchInput big />`)